packages/components/src/InputTime/InputTime.test.tsx

```diff
@@ -20,13 +20,10 @@ it("renders a InputTime", () => {
       <label
         className="label"
         htmlFor="123e4567-e89b-12d3-a456-426655440001"
-      >
-         
-      </label>
+      />
```

Comment:
Any idea why the format of all these `label`s changed? I don't think it's a problem just wondering. 😊

Reply:
I also wondered about this. Im not sure why this changed... I saw this and didn't think it was a big deal. I can investigate more if you would like?

Reply:
I wouldn't worry about it was more just curious. 😂